e2e_playwright/static_app_test.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2025)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from playwright.sync_api import Page, expect
+
+from e2e_playwright.conftest import ImageCompareFunction
+
+
+@pytest.mark.query_param("?staticAppId=Hyperparameter_Tuning_with_sklearn")
```

Comment:
Looks like I forgot we need to register the mark.

See https://docs.pytest.org/en/stable/how-to/mark.html#registering-marks
And our file https://github.com/streamlit/streamlit/blob/develop/lib/pytest.ini